### PR TITLE
Setup deployment to GitHub package registry for sbt artifacts

### PR DESCRIPTION
**Pull request by j4ckofalltrades**
Includes changes in #31 
Refs strongbox/strongbox#1927

### CODE REVIEW

File: hello-strongbox-sbt/project/plugins.sbt [hunk 1]
```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.2")
```
Comment (j4ckofalltrades) [inline hello-strongbox-sbt/project/plugins.sbt:1]: See https://github.com/djspiewak/sbt-github-packages

**Comment by j4ckofalltrades** [comment]
@carlspring Requesting review

Edit: Add support for ci-friendly versions (appending timestamp to package version)

**Review by carlspring — APPROVED** [review]
Thanks! This looks good to me!

@steve-todorov : Could you please have a look as well? Thanks! :)

**Review by steve-todorov — APPROVED** [review]
LGTM! Thank you very much! :)

**Comment by carlspring** [comment]
Thanks for your help! :)

**Pull request merged**